app/js/controllers/products-index-controller.js

```diff
@@ -3,11 +3,69 @@
   .module('empireCo')
   .controller('ProductsIndexController', ["$stateParams", productsIndexController]);
 
+  ProductsIndexController.$inject = {'$http', '$scope'}
 
+  function ProductsIndexController($http, $scope){
```

Comment:
function is referencing the controller ProductsIndexController instead of the function productsIndexContrller.